src/main/java/ch/njol/skript/expressions/ExprFireTicks.java

```diff
@@ -0,0 +1,88 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Fire Time")
+@Description("How much time an entity will be burning for.")
+@Examples({"send \"You will stop burning in %fire time of player%\""})
+@Since("INSERT VERSION")
+public class ExprFireTicks extends SimplePropertyExpression<Entity, Timespan> {
+
+	static {
+		register(ExprFireTicks.class, Timespan.class, "(burn[ing]|fire) time", "entities");
+	}
+
+	@Override
+	@Nullable
+	public Timespan convert(Entity entity) {
+		// Here max is used to avoid an exception being thrown when the fire ticks are negative
```

Comment:
This is a simple comment, and we don't need this comment here, it's well understood why the max method is used here.